src/components/Table/EditColumnsButton.stories.tsx

```diff
@@ -51,8 +52,9 @@ export function EditColumnButtonInAction() {
         <EditColumnsButton
           trigger={{ label: "Columns" }}
           placement="right"
-          columns={tableColumns}
+          allColumns={tableColumns}
           setColumns={setColumns}
```

Comment:
Thought: should we rename `setColumns` to `setSelectedColumns` to match the new `selectedColumns` name? Afaiu that is what happens, i.e. if this component calls `setColumns` it's really changing the incoming `selectedColumns` and not the `allColumns` prop?